### DISCUSSION TITLE
Moving from one laptop to another

### DISCUSSION THREAD

**Discussion by heroineinabook** [Ascoware/get-iplayer-automator#414] [Q&A]
Hi,

I remember a zillion years ago there was instructions to move some of the files from one laptop to another mainly to keep your PVR list. I did some digging around, granted not very hard, and I can't seem to find the instructions on how to do this. Can anyone please point me in that direction? Thanks!

Lisa

**Comment by skovatch** [comment]
Queue.automatorqueue contains your PVR list.
Formats.automatorqueue contains your preferred download formats for BBC.

But the easiest thing to do is to copy the contents of "~/Library/Application Support/Get iPlayer Automator" from the old to the new laptop.

**Comment by heroineinabook** [comment]
Thanks Scott! Much appreciated.
